### PR TITLE
store/tikv: add txn size metrics.

### PR DESCRIPTION
**Pull request by disksing**
Count the size of data that a transaction writes.

### CODE REVIEW

File: store/tikv/client.go [hunk 1]
```diff
@@ -62,7 +62,6 @@ func newRPCClient() *rpcClient {
 
 // SendCopReq sends a Request to co-processor and receives Response.
 func (c *rpcClient) SendCopReq(addr string, req *coprocessor.Request, timeout time.Duration) (*coprocessor.Response, error) {
-	sendReqCounter.WithLabelValues("cop").Inc()
```
Comment (shenli) [inline store/tikv/client.go]: Why remove cop counter?

Comment (disksing) [inline store/tikv/client.go]: We already have a `sendReqHistogram` which creates a counter implicitly, so this counter is redundant.

File: store/tikv/metrics.go [hunk 2]
```diff
@@ -149,6 +141,24 @@ var (
...
 			Help:      "Counter of region errors.",
 		}, []string{"type"})
+
+	txnWriteKVLenHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "tidb",
+			Subsystem: "tikvclient",
+			Name:      "txn_write_kv_len",
+			Help:      "Count of kv pairs to write in a transaction.",
+			Buckets:   prometheus.ExponentialBuckets(1, 2, 15),
```
Comment (ngaut) [inline store/tikv/metrics.go]: 15 is too small.

File: store/tikv/metrics.go [hunk 3]
```diff
@@ -149,6 +141,24 @@ var (
 			Name:      "region_err_total",
 			Help:      "Counter of region errors.",
 		}, []string{"type"})
+
+	txnWriteKVLenHistogram = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "tidb",
+			Subsystem: "tikvclient",
+			Name:      "txn_write_kv_len",
```
Comment (ngaut) [inline store/tikv/metrics.go]: s/txn_write_kv_len/txn_write_kv_count/

File: store/tikv/metrics.go (hunk 2 above)
Comment (disksing) [inline store/tikv/metrics.go]: 2^14 is up to ~16k, should be enough.

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**